pilz_control/test/acceptance_test_speed_monitoring.md

```diff
@@ -0,0 +1,66 @@
+<!--
+Copyright © 2020 Pilz GmbH & Co. KG
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Acceptance Test Speed Monitoring using the real robot
+
+## Prerequisites
+  - Properly connect and startup the robot and power cabinet containing the PSS4000.
+  - Make sure an emergency stop is within reach.
+
+## Start in T1 mode
+
+### Test Sequence
+
+  1. Press the acknowledge button. Make sure the green light on the robot is on and mode T1 is enabled. Run
+  ```
+  roslaunch prbt_moveit_config moveit_planning_execution.launch sim:=False pipeline:=pilz_command_planner
+  ```
+  and
+  ```
+  roslaunch pilz_control joint_states_speed_observer.launch
+  ```
+  2. Execute `rosrun pilz_control acceptance_test_speed_monitoring.py`.
+  3. Perform some movements via RViz. Choose different velocity scales.
```

Comment:
Choose velocity scales (0,1] ?

Reply:
Actually this is a bit to fuzzy for my taste. Are there edge cases that should be checked?

Reply:
Yes, thanks.